planning/stdlib_types/implementation_plan.md

```
@@ -771,6 +771,304 @@ Ensure the current Promise<T, E> behavior continues to work:
+
+    symValue := checker.GlobalScope.GetValue("Symbol")
+    assert.NotNil(t, symValue, "Symbol value should be defined")
+}
+```
+
+#### Expected Result
+
+After implementing Option D:
+
+1. **Dependency graph** remains unchanged - all cyclic types stay in the same SCC:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)